config-tests/test_gpdr_functionality.py

```diff
@@ -0,0 +1,100 @@
+import configparser
```

Comment:
filename is wrong `General Data Protection Regulation` not `General Protection Data Regulation` :smirk: